llvm/lib/DebugInfo/DWARF/DWARFUnit.cpp

```diff
@@ -653,6 +659,10 @@ bool DWARFUnit::parseDWO(StringRef DWOAlternativeLocation) {
 }
 
 void DWARFUnit::clearDIEs(bool KeepCUDie) {
+  // We need to acquire the FreeDIEsMutex lock in write-mode, because we are
```

Comment:
This comment is out of date - the recursive_mutex isn't shared, so there's no "write-mode" at work here, I think?

Reply:
Oops: That slipped through. I'll update it.